README.md

```diff
@@ -362,6 +363,27 @@ MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py build --cmake
 ccmake build  # or cmake-gui build
 ```
 
+##### Installation Issue
+
+Sometimes you would encounter PyTorch local source is not being detected by Python interpreter after building PyTorch above.
```

Comment:
I'm not sure what problem this solves, perhaps we should just make sure in the docs somewhere to tell people to make sure they don't have an existing installation of pytorch and to uninstall it first

Also the right place for these changes is the `CONTRIBUTING.md` not the main `README.md`